jest.config.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
   testRegex: '((\\.|/)(e2e|test|spec))\\.[jt]sx?$',
   testSequencer: require.resolve('./jest.sequencer'),
   moduleNameMapper: {
-    '@defichain/(?!whale-api-client)(.*)': '<rootDir>/packages/$1/src',
+    // '@defichain/(?!whale-api-client)(.*)': '<rootDir>/packages/$1/src',
```

Comment:
Should be a bad merge into base branch, `@defichain/whale-api-client` is not yet refactored into this codebase.